### PR TITLE
Updated requirements and format changelog

### PR DESCRIPTION
**Pull request by zkanda**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-iron-mq-v3
+iron-mq
```
Comment (mknecht) [inline requirements.txt]: why? could you state reasons for such changes in the commit? that'd be very helpful.

Comment (zkanda) [inline requirements.txt]: `iron-mq-v3` doesn't exist.

**Comment by zkanda** [comment]
@mknecht What do you think of this? Close?

**Comment by mknecht** [comment]
let's merge. is still a useful PR. :)

**Pull request merged**